plugins/jobs/src/js/pages/JobTaskDetailPage.js

```diff
@@ -30,6 +59,16 @@ class JobTaskDetailPage extends React.Component {
     ];
 
     const task = MesosStateStore.getTaskFromTaskID(taskID);
+
+    if (!task) {
+      return (
+        <Trans>
+          Either the data related to that task has already been cleaned up or
+          the given task-id does not exist.
```

Comment:
I think that "task-id" should be written without a dash.

Reply:
i made it `Task ID` now as it looks like a proper noun to me